app/src/main/java/com/example/kotlin/movieapp/ui/main/MainViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.kotlin.movieapp.ui.main
+
+import android.app.Application
+import android.arch.lifecycle.ViewModel
+import android.arch.lifecycle.ViewModelProvider
+import android.databinding.ObservableField
+import com.example.kotlin.movieapp.ui.base.BaseViewModel
+import io.reactivex.Observable
+import io.reactivex.disposables.Disposables
+import java.util.concurrent.TimeUnit
+
+class MainViewModel(application: Application) : BaseViewModel(application) {
+    val title = "Counter"
+    val counter: ObservableField<String> = ObservableField()
+    var counterDisposable = Disposables.disposed()
+
+    init {
```

Comment:
I was thinking we don't have to, this is just an example I was trying out. It will be removed when we add real code